ksqldb-api-client/pom.xml

```diff
@@ -194,6 +194,13 @@
                     <notree>true</notree>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Those are the original changes from #7808 (included for not to make the build pass).